tests/providers/google/cloud/operators/test_local_to_gcs.py

```diff
@@ -42,6 +67,10 @@ def setUp(self):
             'start_date': datetime.datetime(2017, 1, 1)
         }
         self.dag = DAG('test_dag_id', default_args=args)
+        with open('/tmp/fake1.csv', 'wb') as f:
+            f.write(b"x" * 393216)
+        with open('/tmp/fake2.csv', 'wb') as f:
+            f.write(b"x" * 393216)
```

Comment:
This files are not delted, thus this test creates side effects. Please use temp files or mocking.

Reply:
Thanks @turbaszek use of temp files has been implemented in the latest.